examples/simple/main.tf

```diff
@@ -49,6 +48,30 @@ module "materialize_infrastructure" {
   enable_monitoring      = true
   metrics_retention_days = 3
 
+  # Enable and configure Materialize operator
+  install_materialize_operator = true
+
+  # Once the operator is installed, you can define your Materialize instances here.
+  # Uncomment the following block (or provide your own instances) to configure them.
+  # materialize_instances = [
+  #   {
+  #     name           = "analytics"
+  #     namespace      = "materialize-environment"
+  #     database_name  = "analytics_db"
+  #     cpu_request    = "2"
+  #     memory_request = "4Gi"
+  #     memory_limit   = "4Gi"
+  #   },
+  #   {
+  #     name           = "production"
+  #     namespace      = "materialize-environment"
+  #     database_name  = "production_db"
```

Comment:
Q: Wondering if naming this "production" might be misleading if we want to ensure that people view these Terraforms as something we provide for evaluation purposes?